core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1705,6 +1705,7 @@ public SqlOperandCountRange getOperandCountRange() {
           OperandTypes.NUMERIC,
           SqlFunctionCategory.NUMERIC);
 
+  public static final SqlFunction SINH = SqlLibraryOperators.SINH;
```

Comment:
is `SINH` part of standard SQL ?

Reply:
> is `SINH` part of standard SQL ?

Actually I want to support standard SQL at the same time.

Reply:
Well, I think if it's not part of standard SQL, it should not be in `SqlStdOperatorTable`

Reply:
well , Let change it.

Reply:
> Well, I think if it's not part of standard SQL, it should not be in `SqlStdOperatorTable`

hi @yanlin-Lynn I have changed it.